### DISCUSSION TITLE
A vertical line, Experimental: Title Bar

### DISCUSSION THREAD

**Discussion by AndrewAung11** [atom/atom#22459] [General]
There is a vertical line in the half of the editor. Only showing up when the file is opened, not showing up when setting page is opened.
![image](https://user-images.githubusercontent.com/64914752/119016372-dd711580-b9bf-11eb-95fc-5a3fef035617.png)

And for the title bar when I change it to hidden it also hide the menu bar - I haven't set it to auto hide and I can't toggle it by pressing `Alt` key.

**Comment by ThatXliner** [comment]
You can turn off the vertical line by searching in the docs. It's called wrap guide.

**Reply by AndrewAung11** [reply]
@ThatXliner Thank you! I should have known it. 😁
